index.js

```diff
@@ -27,24 +27,33 @@ try {
     getMetrics () { return {} }
   }
 }
+
 const Event = module.exports.Event
 
 //
 // augment the c++ event object with a string conversion
 // function.
 //
+
+const validXtrace = (xtrace) => {
+  // https://github.com/librato/trace/tree/master/docs/specs
+  const regExp = /\b2B[0-9A-F]{40}[0-9A-F]{16}0[0-1]{1}\b/
+  const matches = regExp.exec(xtrace)
+
+  return matches ? matches[0] : ''
```

Comment:
Question: Why here return matches[0] or '' but not the same return for `validTraceparent`?

Reply:
Cleaned. Thanks for catching.